.gitignore

```diff
@@ -123,3 +122,5 @@ output-folders-for-testing
 
 # Jest joint reports directory
 test-reports
+# .vscode
+.vscode/launch.json
```

Comment:
Can't this be useful to other contributors (e.g., sensible debug config defaults)?